src/main/java/frc/robot/subsystems/swerve/modules/ModuleUtils.java

```diff
@@ -57,9 +53,14 @@ public static double velocityToOpenLoopVoltage(
      * @param moduleAngle the angle or velocity in angle of the module
      * @return the distance or velocity without the coupling
      */
-    public static double removeCouplingFromRevolutions(Rotation2d drivePosition, Rotation2d moduleAngle, double couplingRatio) {
+    public static Rotation2d removeCouplingFromAngle(Rotation2d drivePosition, Rotation2d moduleAngle, double couplingRatio) {
         double coupledAngle = moduleAngle.getRotations() * couplingRatio;
-        return drivePosition.getRotations() - coupledAngle;
+        return Rotation2d.fromRotations(drivePosition.getRotations() - coupledAngle);
```

Comment:
consider doing either `Rotation2d coupledAngle = moduleAngle.times(couplingRatio);` and `return drivePosition.minus(coupledAngle);`
or
return `drivePosition.minus(Rotation2d.fromRotations(coupledAngle));`

Reply:
The problem with functions of rot2d is that they like to do module stuff on angles even if I don’t want